compiler/optimizer/RematTools.hpp

```diff
@@ -53,20 +53,23 @@ class RematTools
    {
    static TR_YesNoMaybe gatherNodesToCheck(TR::Compilation *comp,
       TR::Node *privArg, TR::Node *currentNode, TR::SparseBitVector &scanTargets,
-      TR::SparseBitVector &symRefsToCheck, bool trace, TR::SparseBitVector &visitedNodes);
+      TR::SparseBitVector &symRefsToCheck, bool trace, TR::SparseBitVector &visitedNodes,
+      bool safeIndirectLoads);
```

Comment:
This should probably go before visitedNodes since it is meant to mirror the public API.